### DISCUSSION TITLE
How can I do this?

### DISCUSSION THREAD

**Discussion by JoseLRiveraCruz** [OrcaSlicer/OrcaSlicer#5701] [General]
Hi, Im new into 3d printing. I bought a transparent ABS filament and I don't like the look of the inside honeycomb structure like. How can I make a solid object on orca slicer? Please let me know. Thanks all

**Comment by LJKraus** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/245dced5-fcda-4045-9c18-eebfdf74f858)
